integration-tests/tests/src/performance-tests/property-reads.ts

```diff
@@ -85,26 +87,26 @@ function describeTypeRead({ type, value, schema = [] }: TestParameters) {
 
 const cases: Array<TestParameters | [Realm.PropertySchemaShorthand | Realm.PropertySchema, Value]> = [
   ["bool", true],
+  [{ type: "bool", optional: false }, true],
   ["int", 123],
+  [{ type: "int", optional: false }, 123],
   ["float", 123.456],
```

Comment:
Could it make sense to add an optional float as well?

Reply:
The difference for `float` is much smaller because it isn't a primitive at the core/JS interface. We need to produce an instance of the `Float` class to distinguish it from `double`. I tried it and it was only ~10% slower rather than ~25%. Also, once my fix goes in, the perf difference for optional will go away, so we don't need a ton of these. I just want a few to detect if a difference ever pops back up, and it might as well be the fastest ones.